lib/tasks/translation_sync.rake

```diff
@@ -0,0 +1,37 @@
+namespace :translations do
+  desc 'Synchronize missing translation keys from en to specified language'
+  task :sync, [:locale] => :environment do |_task, args|
+    source_locale = 'en'
+    target_locale = args[:locale] || 'ar'
```

Comment:
Could we consider aborting instead of defaulting to the `ar` locale and displaying an error message?

Reply:
yes sure